src/schema/submission.yaml

```diff
@@ -1500,40 +1527,77 @@ enums:
     permissible_values:
```

Comment:
We need to convert the permissible values into upper snake case.

Reply:
I opened a Github issue for that